plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2460,3 +2460,62 @@ procedure.common.no_procedure=(no procedure)
 procedure.common.fixed=Fixed: 
 procedure.common.if=\ if: 
 procedure.common.do=\ do: 
+gradle.errors.title=Gradle task failed with error
+gradle.errors.error=ERROR CODE: 
+gradle.errors.options.do_nothing=Do nothing
+gradle.errors.options.clear_caches=Clear Gradle caches
+gradle.errors.open_options=Open Gradle options
+gradle.errors.open_help_page=Open help page
+gradle.errors.cache_corrupted=<html><big>Gradle caches are corrupted!</big><br>\
+  <br>It seems that the cached dependencies of Gradle got corrupted or are outdated.<br>\
+  Cache needs to be cleared and re-downloaded. You can do this by pressing the button below.<br>\
+  <br>After you clear cache, the first Gradle build could take a but longer than usual.
+gradle.errors.cache_outdated=<html><big>Gradle caches are outdated!</big><br>\
+  <br>You are using MCreator in offline mode. Caches got outdated and need to be updated.<br>\
+  You need to disable offline mode in MCreator's preferences under Gradle options section<br>\
+  so the Gradle can download new version of Gradle dependencies and cached files.<br>\
+  <br>Afterwards, you can turn the offline mode back on.
+gradle.errors.invalid_java_version=<html><big>Invalid Java version is in use!</big><br>\
```

Comment:
Keep here Java 8 as this error can only happen with java 8. Maybe in the future generators will use own java 8 if they needed to so keep the original version numbers.